special-use/special-use-suds-mapping-get.json

```diff
@@ -0,0 +1,82 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Special use application",
+    "description": "A rough sketch of the JSON response to a GET HTTP request for a special use permit from our ePermitting system for the U.S. Forest Service",
+    "definitions": {
+      "applicant-info": {
+        "type": "object",
+        "properties": {
+          "contactControlNumber": { "type": "string", "basic-api-field" : "CONT_CN"},
+      		"firstName": { "type": "string", "basic-api-field" : "FIRST_NAME"},
+      		"lastName": {	"type": "string", "basic-api-field" : "LAST_NAME" },
+          "dayPhone": { "$ref": "#/definitions/phone-number" },
+          "eveningPhone": { "$ref": "#/definitions/phone-number" },
+          "emailAddress": { "type": "string", "basic-api-field" : "EMAIL"},
+          "mailingAddress": { "type": "string", "basic-api-field" : "ADDRESS_1"},
+          "mailingAddress2": { "type": "string", "basic-api-field" : "ADDRESS_2"},
+          "mailingCity": { "type": "string", "basic-api-field" : "CITY_NAME"},
+          "mailingState": { "type": "string", "basic-api-field" : "STATE_CODE"},
+          "mailingZIP": { "type": "integer", "basic-api-field": "POSTAL_CODE"},
+      		"organizationName": {
+      			"description": "The name of the group, company, organization or event applying",
+      			"type": "integer",
+      		},
+          "website": { "type": "string", "suds-table": "Contact", "suds-field": "Remarks" },
```

Comment:
I need to double check this, but I don't think there is a SUDS field for website address at the moment (even though it's on the form), so frontliners put it into the remarks field. :(